bedrock/firefox/templates/firefox/new/scene1.html

```diff
@@ -61,6 +69,12 @@
               <div class="mobile download-button-wrapper" id="download-button-wrapper-mobile" data-upgrade-subtitle="{{_('Get it free on Google Play')}}">
                 {{ download_firefox(dom_id='download-button-mobile') }}
               </div>
+
+            {% if version == '2' %}
+              <div id="test-inpage-wrapper">
```

Comment:
Ah, yeah, tested in Beta and Dev Edition. 😞 Moving this block to the last element in `main-content` seems to do the trick.